openage/convert/slp.pyx

```diff
@@ -405,12 +526,12 @@ cdef class SLPFrame:
                 dpos += 1
                 color = self.get_byte_at(dpos)
 
-                # TODO: verify this. might be incorrect.
-                # color = ((color & 0b11001100) | 0b00110011)
-
-                # SpecialColor class preserves the calculation of
-                # player*16 + color
                 for _ in range(cpack.count):
+                    # TODO: verify this. might be incorrect.
+                    # color = ((color & 0b11001100) | 0b00110011)
```

Comment:
i think we can remove that. for the main sprites we have the correct palette index already, so this might be for some other encoding case?